lms/envs/test.py

```diff
@@ -381,6 +381,22 @@
     'django.core.files.uploadhandler.TemporaryFileUploadHandler',
 )
 
+###################### Grade Downloads ######################
+# When tests run concurrently, make sure that they do not
+# interfere with each other when creating or removing temp files.
+GRADES_DOWNLOAD = {
+    'STORAGE_TYPE': 'localfs',
+    'BUCKET': 'edx-grades',
+    'ROOT_PATH': '/tmp/edx-s3-{}/grades'.format(THIS_UUID),
```

Comment:
Oh, snaps. Good call. I added some debug printing and as you suspected, os.pid in the settings file is only evaluated once, not re-evaluated for each forked subprocess.